www/layouts/partials/head/css.html

```diff
@@ -1,12 +1,14 @@
-{{- with resources.Get "css/theme.css" }}
-  {{- if eq hugo.Environment "development" }}
-    <link rel="stylesheet" href="{{ .RelPermalink }}">
-  {{- else }}
-    {{- with . | minify | fingerprint }}
-      <link rel="stylesheet" href="{{ .RelPermalink }}" integrity="{{ .Data.Integrity }}" crossorigin="anonymous">
+{{ range $css := slice "css/theme.css" "css/icons.css" }}
```

Comment:
This change is here to also load the css/icons.css stylesheet. Feel free to integrate it otherwise obviously.